sdk/cosmosdb/cosmos/package.json

```diff
@@ -11,98 +13,121 @@
-    "check-format": "prettier --list-different --config .prettierrc.json \"src/**/*.ts\" \"test/**/*.ts\"",
-    "compile": "echo Using TypeScript && tsc --version && tsc -b --pretty",
-    "compile:samples": "tsc -b samples",
-    "docs": "typedoc --tsconfig ./src/tsconfig.json --excludePrivate --mode file --out ./dist/docs ./src",
+    "audit": "node ../../../common/scripts/rush-audit.js && rimraf node_modules package-lock.json && npm i --package-lock-only 2>&1 && npm audit",
+    "build:samples": "tsc -b samples",
+    "build:src": "echo Using TypeScript && tsc --version && tsc -b --pretty",
+    "build:test": "tsc -b src test --verbose",
+    "build": "npm run clean && npm run extract-api && npm run bundle && node writeSDKVersion.js",
     "bundle": "rollup -c",
     "bundle-types": "node bundle-types.js",
-    "build": "npm run clean && npm run check-format && npm run lint && npm run compile && node writeSDKVersion.js && npm run docs && npm run bundle && npm run bundle-types",
-    "test": "tsc -b src test --verbose && mocha --reporter mocha-multi --reporter-options spec=-,mocha-junit-reporter=- -r esm -r dotenv/config -r ./test/common/setup.js \"./test/**/*.spec.js\" --timeout 100000",
-    "prepack": "npm install && npm run build",
+    "check-format": "prettier --list-different --config ../../.prettierrc.json \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\"",
+    "clean": "rimraf dist dist-esm temp types *.tgz *.html *.log *.tsbuildinfo test/**/*.{js,js.map,d.ts}",
+    "docs": "typedoc --tsconfig ./src/tsconfig.json --excludePrivate --mode file --out ./dist/docs ./src",
+    "extract-api": "npm run build:src && api-extractor run --local",
+    "format": "prettier --write --config ../../.prettierrc.json \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\"",
+    "integration-test:browser": "echo skipped",
+    "integration-test:node": "mocha -r esm -r dotenv/config -r ./test/common/setup.js \"./test/**/*.spec.js\" --timeout 100000",
+    "integration-test": "npm run integration-test:node && npm run integration-test:browser",
+    "lint:fix": "eslint \"src/**/*.ts\" \"test/**/*.ts\" -c ../../.eslintrc.json --fix --fix-type [problem,suggestion]",
+    "lint": "eslint -c ../../.eslintrc.json src test --ext .ts -f html -o cosmos-lintReport.html || exit 0",
+    "pack": "npm pack 2>&1",
+    "prebuild": "npm run clean",
+    "test:browser": "npm run build:test && npm run unit-test:browser && npm run integration-test:browser",
+    "test:node": "npm run build:test && npm run unit-test:node && npm run integration-test:node",
     "test-browser": "karma start ./karma.config.js --single-run",
-    "test-consumer": "node consumer-test.js"
+    "test-consumer": "node consumer-test.js",
+    "test": "npm run build:test && npm run unit-test && npm run integration-test",
+    "unit-test:browser": "echo skipped",
+    "unit-test:node": "echo skipped",
+    "unit-test": "npm run unit-test:node && npm run unit-test:browser"
+  },
+  "dependencies": {
```

Comment:
These should all be caret versions.  We can fix this in a later PR.

Reply:
Sure, upcoming PR for this.